tests/mir_test_framework/fake_input_device_impl.cpp

```diff
@@ -324,6 +324,9 @@ void mtf::FakeInputDeviceImpl::InputDevice::map_touch_coordinates(float& x, floa
 
 void mtf::FakeInputDeviceImpl::InputDevice::start(mi::InputSink* destination, mi::EventBuilder* event_builder)
 {
+    // We keep track of events based on their timestamps, so we can't have those being modified
+    event_builder->calibrate_timestamps(false);
+
```

Comment:
Is this the only use of `calibrate_timestamps()`? Wouldn't it be better to supply a test double as the clock?

Reply:
That's a little inconvenient, because the code where it's obvious that we need to do this doesn't create the builder.

Reply:
It doesn't create the builder passed to `start()`, but it _could_ create the builder used by `FakeInputDeviceImpl`.

I'm not sure it is worth the effort, but I'd like clarity on the purpose of this method.